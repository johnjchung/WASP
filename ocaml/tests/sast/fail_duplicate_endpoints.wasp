Endpoint e1(GET:) {
	float x = 3.3;
}

Endpoint e1(GET:) {
	string y = "hey";
}
