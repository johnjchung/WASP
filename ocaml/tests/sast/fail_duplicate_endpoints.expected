Fatal error: exception Symbol_table.DuplicateObjFuncError("YOU CANNOT DECLARE TWO OBJECTS, ENDPOINTS, OR FUNCS IN THE SAME PROGRAM WITH THE SAME NAME: e1")
